docs/installguide/system_requirements.rst

```diff
@@ -1,12 +1,87 @@
 System requirements
 ===================
 
+Operating systems
+-----------------
+
+ * Windows XP, Vista, 7, 8, 10
+ * Mac OSX 10.9, 10.10 and 10.11
+ * Linux: Any system with Python 2.7
+ * Debian/Raspberry Pi packages: Wheezy or later
+ * Ubuntu packages: 14.04, 15.10, 16.04 - anything that's *not* end-of-life.
+
+
 Supported Browsers
 ------------------
-KA Lite is currently *not* supported on Internet Explorer version 8 or lower. You must use IE9 or later.
 
-You can also use KA Lite on browsers like Firefox, Chrome, and Safari!
+ * IE9+
+ * Firefox
+ * Chrome
+ * Safari
+
+KA Lite is currently *not* supported on Internet Explorer version 8 or lower.
+
+Known issues:
+
+ * Videos do not play on Windows XP (use the Chrome browser)
+ * Subtitles do not work for Epiphany (the browser in Raspberry Pi).
+
+
+Video playback
+--------------
+
+Videos are MP4 encoded. On Ubuntu/Debian systems, install the `Ubuntu restricted extras package <https://apps.ubuntu.com/cat/applications/ubuntu-restricted-extras/>`_.
+
+
+Hardware requirements
+---------------------
+
+Clients
+^^^^^^^
+
+KA Lite is running as a client on very old desktops and very low-scale
```

Comment:
"running as a client" may be confusing. Perhaps "Very old desktops and very low-power computers can be used as client devices to access KA Lite."